src/test/java/poke/fromitive/attendance/BaseSpringBootTest.java

```diff
@@ -0,0 +1,15 @@
+package poke.fromitive.attendance;
+
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
```

Comment:
`ActiveProfiles`를 사용하지 않고 `SpringBootTest`의 `classes`로 import해오는 건 어떤가요?

```suggestion
@SpringBootTest(
        webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
        classes = TestConfig.class
)

Reply:
`TestConfiguration.java`를 `TestConfig.java`로 바꾼 뒤, `@TestConfiguration`을 붙이면 해결됩니다

Reply:
`AttendanceServiceTest.java` 테스트를 디버깅 해보니까 `TestConfig`를 타지 않고 `GoogleCrawlerConfiguration` 설정에 있는 값을 주입받네요 😢

Reply:
우선 테스트 빈쪽에 `@Primary`를 붙여보죠!
```java
    @Bean
    @Primary
    AttendanceCrawler testAttendanceCrawler() {
        return new FakeAttendanceCrawler();
    }
```